Chapter_08/8-7.py

```diff
@@ -0,0 +1,18 @@
+def make_album(artist_name, album_title, songs=None):
+    """Stores album information in a dictionary"""
+    album = {
+        'arist': artist_name,
+        'album': album_title,
+    }
+    if songs:
+        album['songs'] = songs
+    return album
+
+album = make_album('Boris Slavov', "Baldur's Gate 3 Original Soundtrack",)
+print(album)
+
+album = make_album('Olivia Rodrigo', 'guts', 12)
+print(album)
+
+album = make_album('Noah Kahan', 'Stick Season')

```

Comment:
Opted against these changes since it's where I was at that point in time. I think now I'm confident enough to do it the more concise way the first try lol